packages/components-react/src/pages/Table.tsx

```diff
@@ -1,14 +1,18 @@
 import { PTable, PTableHead, PTableHeadCell, PTableHeadRow } from '@porsche-design-system/components-react';
-import { TableWithCaption } from '../components';
+import { TableWithCaptionDark, TableWithCaption } from '../components';
 
 export const TablePage = (): JSX.Element => {
   return (
     <>
-      <div className="playground light" title="should render table">
+      <div className="playground light" title="should render table with light mode">
```

Comment:
The caption seems to be missing in the result?
<img width="1114" alt="Bildschirm­foto 2023-03-03 um 10 02 37" src="https://user-images.githubusercontent.com/4380295/222678272-cdf34bc2-656e-4ce8-98e1-5a193cad7189.png">